.pipelines/prchecks/PackageBuildPRCheck.yml

```diff
@@ -97,6 +97,7 @@ extends:
                       outputArtifactsFolder: $(ob_outputDirectory)
                       pipArtifactFeeds: "mariner/Mariner-Pypi-Feed"
                       selfRepoName: self
+                      testSuiteName: "[${{ configuration.name }}] Package test"
```

Comment:
Can you add the architecture to the name as well? (it seems your pipeline PR has the architecture in the testSuiteName)

Reply:
The `configuration.name` is actually the architecture.;)